views/js/review/plugins/navigation/review-panel/plugin.js

```diff
@@ -13,7 +13,7 @@
  * along with this program; if not, write to the Free Software
```

Comment:
Please update bundle` views/js/loader/ltiTestReview.min.js` - it's not latest version (unless it's left for release action)